provider/cldr/src/transform/list/mod.rs

```diff
@@ -2,6 +2,7 @@
 // called LICENSE at the top level of the ICU4X source tree
```

Comment:
Should this be `list.rs` instead of `list/mod.rs`, as there's only a single file in the module? Same for plurals

Reply:
Good question; I made them all folders because:

1. More consistent
2. Easier to refactor code and create sub-modules in the future

In other words, I see advantages to doing `list/mod.rs` with few disadvantages.